main.go

```diff
@@ -31,11 +35,18 @@ func main() {
 	var fnames, dirnames []string
 	content := make(map[string]_escFile)
 	prefix := filepath.ToSlash(*flagPrefix)
+	var ignoreRegexp *regexp.Regexp
+	if *flagIgnore != "" {
+		ignoreRegexp = regexp.MustCompile(*flagIgnore)
```

Comment:
MustCompile panics if it fails. Let's use the normal parsing and log.Fatal(err) on regexp parse failure instead.